.eslintignore

```diff
@@ -0,0 +1,2 @@
+source/javascripts/**/*.js
+source/javascripts/**/*.js.erb
```

Comment:
this is mainly for the IDE, the CI does not do linting of tests (but coming soon ;))